source/uri-options/uri-options.rst

```diff
@@ -210,6 +210,12 @@ pertaining to URI options apply here.
      - required for drivers with connection pools
      - The maximum number of clients or connections able to be created by a pool at a given time. This count includes connections which are currently checked out.
 
+   * - maxConnecting
+     - positive integer
+     - defined in the `Connection Pooling spec`_
+     - required for drivers with connection pools
+     - The maximum number of Connections a Pool may be establishing concurrently.
```

Comment:
Please add spec tests to check that only positive integers are valid for `maxConnecting`.

Reply:
Done, see `connection-pool-options.yml`.